tests/regression/test_gate_seal.py

```diff
@@ -0,0 +1,166 @@
+    assert contracts.validators_exit_bus_oracle.getResumeSinceTimestamp() == seal_tx.timestamp + DURATION_SECONDS
+
+    # reverts on requestWithdrawals
+    contracts.lido.approve(contracts.withdrawal_queue.address, REQUESTS_SUM, {"from": steth_holder})
+    with reverts("typed error: 0x14378398"):  # signature for ResumedExpected()
+        contracts.withdrawal_queue.requestWithdrawals(
+            [REQUEST_AMOUNT for _ in range(REQUESTS_COUNT)], steth_holder, {"from": steth_holder}
+        )
+
+    # reverts on VEBO report
+    with reverts("typed error: 0x14378398"):  # signature for ResumedExpected()
+        contracts.validators_exit_bus_oracle.submitReportData((1, 1, 1, 1, ZERO_BYTES32), 1, {"from": steth_holder})
+
+    # reverts on finalization attempt
+    with reverts("typed error: 0x14378398"):  # signature for ResumedExpected()
+        contracts.withdrawal_queue.finalize(1, 1, {"from": steth_holder})
+
+    """ claim """
+    lastCheckpointIndex = contracts.withdrawal_queue.getLastCheckpointIndex()
+    hints = contracts.withdrawal_queue.findCheckpointHints(claimable_request_ids, 1, lastCheckpointIndex)
+    claim_balance_before = account.balance()
+    claim_tx = contracts.withdrawal_queue.claimWithdrawals(claimable_request_ids, hints, {"from": steth_holder})
+    claim_balance_after = account.balance()
+
+    assert almostEqEth(
+        claim_balance_after - claim_balance_before + claim_tx.gas_used * claim_tx.gas_price, REQUESTS_SUM
+    )
+    assert claim_tx.events.count("WithdrawalClaimed") == REQUESTS_COUNT
+
+    """ accounting oracle reports until we pass pause and claim"""
+    (
+        SLOTS_PER_EPOCH,
+        SECONDS_PER_SLOT,
+        _,
+        _,
+        EPOCHS_PER_FRAME,
+        _,
+    ) = get_time_config()
+    MAX_REPORTS_UNTIL_RESUME = DURATION_SECONDS // (SECONDS_PER_SLOT * SLOTS_PER_EPOCH * EPOCHS_PER_FRAME) + 2
+    reports_passed = 0
+    while True:
+        (report_tx, _) = oracle_report(silent=True)
+        reports_passed += 1
+        print(
+            f"Oracle report {reports_passed} at {report_tx.timestamp}/{seal_tx.timestamp + DURATION_SECONDS} seconds to resume"
+        )
+        if report_tx.events.count("WithdrawalsFinalized") == 1:
+            break
+        assert reports_passed <= MAX_REPORTS_UNTIL_RESUME
+
+    """ post seal claim """
+    lastCheckpointIndex = contracts.withdrawal_queue.getLastCheckpointIndex()
+    hints = contracts.withdrawal_queue.findCheckpointHints(pending_request_ids, 1, lastCheckpointIndex)
+    claim_balance_before = account.balance()
+    claim_tx = contracts.withdrawal_queue.claimWithdrawals(pending_request_ids, hints, {"from": steth_holder})
+    claim_balance_after = account.balance()
+
+    assert almostEqEth(
+        claim_balance_after - claim_balance_before + claim_tx.gas_used * claim_tx.gas_price, REQUESTS_SUM
+    )
+    assert claim_tx.events.count("WithdrawalClaimed") == REQUESTS_COUNT
```

Comment:
I would show that both contracts are still paused (like a cherry on a cake)
i.e., highlight that the claim hasn't been prohibited by GateSeal